benchmarks/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pytest
+pytest-benchmarks
```

Comment:
Let's add this line to the `dev` group in `pyproject.toml`. (Just added in #130, #131) and remove the need for a separate requirements file here.